packages/client/components/TeamPrompt/TeamPromptDiscussionDrawer.tsx

```diff
@@ -181,7 +181,7 @@ const TeamPromptDiscussionDrawer = ({meetingRef, onToggleDrawer}: Props) => {
           header={
             <DiscussionHeaderWrapper>
               <PromptResponseEditor content={contentJSON} readOnly={true} />
-              <StyledReactjis reactjis={reactjis} onToggle={onToggleReactji} />
+              <StyledReactjis key={localStageId} reactjis={reactjis} onToggle={onToggleReactji} />
```

Comment:
this is the actual fix. we use `reactji` id as a key, which is the same for all the discussions. adding key here, makes sure reactji section is user specific